content/trusted-content/official-images/using.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Using Docker Official Images
+description: |
+  Learn about building applications with Docker Official images
+  and how to interpret the tag names they use.
+keywords: docker official images, doi, tags, slim, feedback, troubleshooting
+weight: 10
+---
+
+Docker recommends you use the Docker Official Images in your projects.
+These images have clear documentation, promote best practices, and are regularly updated.
+Docker Official Images support most common use cases, making them perfect for new Docker users.
+Advanced users can benefit from more specialized image variants as well as review Docker Official Images as part of your `Dockerfile` learning process.
+
+## Tags
+
+The repository description for each Docker Official Image contains a
+**Supported tags and respective Dockerfile links** section that lists all the
+current tags with links to the Dockerfiles that created the image with those
+tags. The purpose of this section is to show what image variants are available.
+
+![Example: supported tags for Ubuntu](../images/supported_tags.webp)
+
+Tags listed on the same line all refer to the same underlying image. Multiple
+tags can point to the same image. For example, in the previous screenshot taken
+from the `ubuntu` Docker Official Images repository, the tags `24.04`,
+`noble-20240225`, `noble`, and `devel` all refer to the same image.
+
+The `latest` tag for a Docker Official Image is often optimized for ease of use
+and includes a wide variety of useful software, such as developer and build tools.
+Because of their ease of use and wide applicability, `latest` images are often
+used in getting-started guides, but they're typically not optimized for production use.
```

Comment:
The way I usually explain to maintainers for how to choose which variant to put `latest` on is "Which image would you suggest users should use if they have no idea or are unfamiliar with the underlying software?" (which I think more clearly highlights why `ubuntu:latest` is the latest LTS, not the latest release :eyes:)